lib/ansible/plugins/lookup/oc.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+# (c) 2017, Kenneth D. Evensen <kevensen@redhat.com>
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: ocp
+    version_added: "2.5"
+    short_description: Returns the JSON definition of an object in OpenShift
+    description:
+        - The Ansible OC module performs Create, Update, and Delete operations for
```

Comment:
this is not a module ... also lookups should not create/update/delete anything, just query

Reply:
Right.  I was pointing out that the OC Ansible module does the Create, Update and Delete.  This lookup does the Read for CRUD.  If this is misleading or unnecessary I'll remove it.